keystone-init/values.yaml

```diff
@@ -4,6 +4,11 @@
 keystone_init:
   name: keystone-init
 
+  # an optional preexisting serviceAccount to use
+  # to create a service account with the deployment,
+  # deploy with rbac.create=true
+  serviceAccount: ''
```

Comment:
Why define it twice? Would you want to have two different service accounts?

Reply:
The cleanup job and keystone-init itself both need different sets of permissions so there should always be two separate accounts.

For our deployments we're only pre-creating a service account for the cleanup hooks (solution 2 per the pr description), but I don't think it's unreasonable to want to manually create both and skip our serviceaccounts/roles/rolebindings altogether.